### ISSUE TITLE
iOS12 - iPad Landscape it is not working

### ISSUE THREAD

**Issue by Shenbagaganez** [vermont42/JFATabBarController#8]
In our iPad, we have created this TabBar and Pushed to the Login view navigation controller. It is working fine upto iOS11 devices and all the simulators including iOS12 simulators. But is not working in iOS12 device Landscape mode. More button is coming, when we go to portrait and coming back to Landscape it is working. Can you please help me on this.

**Comment by vermont42** [comment]
@Shenbagaganez Are you seeing this bug on device but not in simulator? I do not see it in simulator.

![simulator screen shot - ipad pro 9 7-inch - 2018-11-08 at 07 07 36](https://user-images.githubusercontent.com/3514087/48207248-1b881b80-e325-11e8-9227-2891ff1ee838.png)

I have not attempted embedding `JFATabBarController` in a `UINavigationController`, which it sounds like you are doing.

Please share a video of your attempted usage. This may allow me to reproduce the bug.

**Comment by Shenbagaganez** [comment]
Yes we have issue only on physical device. When we login in Landscape mode more tab bar item is coming instead of custom tab bar.
